### PR TITLE
Improve font size documentation

### PR DESCRIPTION
**Pull request by melimore86**
#3458 issue, adding font size units in R and documentation

### CODE REVIEW

**Comment by thomasp85** [comment]
In the future, please use a descriptive title for your PRs as it makes it much easier to browse🙂

File: R/theme-defaults.r [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #' [theme()] if you just need to tweak the display of an existing
 #' theme.
 #'
-#' @param base_size base font size
+#' @param base_size base font size; text size in pts.
```
Comment (thomasp85) [inline R/theme-defaults.r]:
```suggestion
#' @param base_size base font size, given in pts.
```

**Comment by thomasp85** [comment]
Can I get you to merge in master so that the checks pass?

**Comment by thomasp85** [comment]
@melimore86 do you want to finish this off?

**Comment by clauswilke** [comment]
Thanks, this will be part of the next ggplot2 release.

**Pull request merged**